rpm/maliit-framework-wayland.spec

```diff
@@ -126,9 +127,9 @@ sed -i -e 's:@PATH@:%{_libdir}/maliit-framework-tests:g' %{buildroot}/opt/tests/
 %{_libdir}/libmaliit-plugins.so
 %{_includedir}/maliit-2/
 %{_libdir}/pkgconfig/*.pc
-%{_datadir}/qt5/mkspecs/features/*
+%{_libdir}/qt5/mkspecs/features/*
```

Comment:
Doesn't work in sailfish. There are some PRs for adjusting the paths, but until those are in this should be as it was.

Reply:
Okay, undoing that - I had issues with building that locally

Reply:
NOTE: I was unable to build the rpms by using `sfdk --specfile rpm/maliit-framework-wayland.spec build`, I had to make the change, in the PR however I've edited that back.